rpcs3/Emu/RSX/VK/VKDraw.cpp

```diff
@@ -298,22 +299,23 @@ void VKGSRender::load_texture_env()
 			(update_framebuffer_sourced && vs_sampler_state[i]->upload_context == rsx::texture_upload_context::framebuffer_storage))
 		{
 			auto sampler_state = static_cast<vk::texture_cache::sampled_image_descriptor*>(vs_sampler_state[i].get());
+			auto texture = rsx::method_registers.vertex_textures[i];
```

Comment:
See previous. Also applies to other similar places.

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/f7c2dcd6ac1e185d2745ab681e12225f8ea2f9d5